src/components/PredefinedButtons.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../components/Button';
```

Comment:
You can inline the `Button` class, remove the `Button.js` file and rename this file to `Button.js`.